_pages/en/web-services-access.md

```diff
@@ -99,17 +149,12 @@ Should you approve, the IT Strategy team will work with teams in IITB to:
 
 Note: Inconsistent browser configurations (Websockets, 3rd party cookies blocked), SSH blocked
 
-### Collaboration
+### Document Editing
 
 |             | URL           | Notes  |
 | ----------- |:-------------:| ------:|
-| Doodle | https://www.doodle.com |  |
-| Eventbrite | https://www.eventbrite.ca |  |
 | Framapad |  https://framapad.org |  |
 | Office365 (pilot) | https://www.office.com |  |
-| Slido | https://www.sli.do |  |
```

Comment:
Is Slido a Document Editing SaaS? (It's also listed under **Other**)

Reply:
You are right, fixed :)